docs/general/ddl/alter-table.rst

```diff
@@ -32,6 +32,10 @@ In order to set a parameter to its default value use ``reset``::
     cr> alter table my_table reset (number_of_replicas);
     ALTER OK, -1 rows affected (... sec)
 
+.. NOTE::
+    Changing compression using the ``codec`` parameter only takes effect for
+    newly written segments. To enforce a rewrite of already existing segments,
+    run :ref:`sql-optimize` with the parameter ``max_num_segments = 1``.
```

Comment:
I think a newline is needed after the admonition identifier. Not 100% sure though.

Reply:
GitHub renders it differently than Sphinx, but there _is_ a difference: https://gist.github.com/amotl/3c6085f1b0972a826a3fa7dddd94d5e2

Reply:
I tested it locally using `./gradlew :blackbox:developDocs`, appears to render fine:
<img width="881" alt="Screenshot 2022-06-22 at 08 21 31" src="https://user-images.githubusercontent.com/218003/174957589-1ae63a61-8b7c-4616-afe7-3acfc4259429.png">